### PR TITLE
I appreciate your taste in movies.

### PR DESCRIPTION
**Pull request by uglyprincess**
1 

#### Short description of what this resolves:
This change aims to resolve the issue of changing a .csv file into a .json file

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

File: Submissions/uglyprincess/task_1/task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import csv, json 
+csvFilePath = 'C:/Users/Aaryan/Desktop/Monty_Python_Class/Task_1/movies.csv'
+jsonFilePath = 'movies.json'
+
+data = {}
+
+with open(csvFilePath, encoding = 'utf-8') as csvFile:
+	csvReader = csv.DictReader(csvFile)
+	print(csvReader)
```
Comment (deepakDOTexe) [inline Submissions/uglyprincess/task_1/task1.py]: Avoid printing unnecessary data.

File: Submissions/uglyprincess/task_1/task1.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	for csvflow in csvReader:
+		movieId = csvflow["movieId"]
+		data[movieId] = csvflow
+		
+print(data)
```
Comment (deepakDOTexe) [inline Submissions/uglyprincess/task_1/task1.py:12]: Avoid printing unnecessary data.

File: Submissions/uglyprincess/task_1/task1.py (hunk 1 above)
Comment (uglyprincess) [reply Submissions/uglyprincess/task_1/task1.py]: Sorry sir, I'd just written that line to make sure I was getting the required output, forgot to delete it! 😅

File: Submissions/uglyprincess/task_1/task1.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+jsonFilePath = 'movies.json'
+
+data = {}
+
+with open(csvFilePath, encoding = 'utf-8') as csvFile:
+	csvReader = csv.DictReader(csvFile)
+	print(csvReader)
+	for csvflow in csvReader:
+		movieId = csvflow["movieId"]
+		data[movieId] = csvflow
```
Comment (deepakDOTexe) [inline Submissions/uglyprincess/task_1/task1.py]: You have to create the json file containing only comedy movies.

Comment (uglyprincess) [reply Submissions/uglyprincess/task_1/task1.py]: Okay sir

File: Submissions/uglyprincess/task_1/task1.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+		
+print(data)
+
+
+with open(jsonFilePath, 'w') as jsonFile:
+	jsonFile.write(json.dumps(data, indent=4))
+
```
Comment (deepakDOTexe) [inline Submissions/uglyprincess/task_1/task1.py:20]: Also state what changes u have made in the commit message briefly and maintain the professional decorum of creating PRs.

**Review by deepakDOTexe — CHANGES_REQUESTED** [review]

File: Submissions/uglyprincess/task_1/task1.py (hunk 4 above)
Comment (uglyprincess) [reply Submissions/uglyprincess/task_1/task1.py:20]: Noted sir

**Comment by deepakDOTexe** [comment]
@uglyprincess You have one more day to have this PR merged. After that I will close it due to no activity from you.

**Pull request closed without merging**